pubnub/lib/src/dx/push/push.dart

```diff
@@ -33,7 +37,7 @@ mixin PushNotificationDx on Core {
     if (gateway == PushGateway.apns2) Ensure(topic).isNotNull('topic');
 
     var params = ListPushChannelsParams(keyset, deviceId, gateway,
-        topic: topic, environment: environment);
+        topic: topic, environment: environment, start: start, count: count);
```

Comment:
The same nitpick here, you can add a trailing comma , after `count` so your IDE should break the line